batch-processing/README.md

```diff
@@ -0,0 +1,379 @@
+# Batch Processing
+
+In this project, you'll build a simple image processing pipeline: reading from a CSV file, downloading an image, processing it, and uploading it to cloud storage.
+
+> ⚠️ This project requires you to have access to an Amazon AWS account, with permissions to configure IAM and S3. Ask on CYF Slack for help with that.
+
+> ⚠️ You need a GitHub repo to complete this because we'll be using [GitHub Actions](https://docs.github.com/en/actions).
+
+Learning objectives:
```

Comment:
I'd also maybe add an LO (which I really like that we're doing) which is how to read/modify/extend existing code. Super common software skill which I'm not sure they'll have really done much before. I really like that this is doing that!